NEWS.md

```diff
@@ -1,5 +1,10 @@
 # readr 0.2.2.9000
 
+* Named lists can now be used as input to `col_types()` (#401, @jimhester).
+
+* Column specifications are now printed by default when reading, and can be
```

Comment:
I think I have been simply putting the issue number rather than the PR number in NEWS, as I don't always know the PR number if I haven't opened it yet.